conans/test/unittests/util/apple_test.py

```diff
@@ -82,40 +85,3 @@ def test_to_apple_arch(self):
         self.assertEqual(_to_apple_arch('armv8_32'), 'arm64_32')
         self.assertIsNone(_to_apple_arch('mips'))
         self.assertEqual(_to_apple_arch('mips', default='mips'), 'mips')
-
-    @pytest.mark.skipif(platform.system() != "Darwin", reason="Requires OSX")
-    def test_xcrun(self):
```

Comment:
Copied to integration tests. It's running xcrun tool so, IMO, better to keep it there.